recipes/firefox.rb

```diff
@@ -43,6 +43,23 @@
   recursive true
 end
 
+# Create dedicated log directory
+directory node['aet']['firefox']['log_dir'] do
+  owner node['aet']['firefox']['user']
+  group node['aet']['firefox']['group']
+  mode '0755'
+  action :create
+  recursive true
+end
+
+
+# Create stderr.log file in log directory
+file "#{node['aet']['firefox']['log_dir']}/stderr.log" do
```

Comment:
Is this required element? I believe firefox will create the file on it's own

Reply:
I cannot see such file on my instance, where should it be located?